deltacat/utils/daft.py

```diff
@@ -22,6 +21,35 @@
 logger = logs.configure_deltacat_logger(logging.getLogger(__name__))
 
 
+def _get_compatible_target_schema(
```

Comment:
So `pyarrow.py` depends on `daft.py`. Importing from `pyarrow.py` here would cause a circular dependency.

I can maybe create a common `schema.py` utility file that both can inherit from? Lmk what you think.

Reply:
Yeah, that sounds good.

Reply:
I moved the utility function back to `pyarrow.py`, since the behavior is now different in `daft.py` and we no longer need to share this utility